src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/SignedHttpRequestHandler.cs

```diff
@@ -623,8 +645,9 @@ internal virtual async Task<SecurityKey> ValidateSignatureAsync(JsonWebToken sig
         /// <param name="signedHttpRequest">A SignedHttpRequest.</param>
         /// <param name="signedHttpRequestValidationContext">A structure that wraps parameters needed for SignedHttpRequest validation.</param>
         /// <remarks>
-        /// This method will be executed only if <see cref="SignedHttpRequestValidationParameters.ValidateTs"/> is set to <c>true</c>.
-        /// </remarks>    
+        /// This method will be executed only if <see cref="SignedHttpRequestValidationParameters.ValidateTs"/> is set to <c>true</c> or a ts claim is present and
+        /// <see cref="SignedHttpRequestValidationParameters.ValidatePresentClaims"/> is set to true.
```

Comment:
Might be helpful to include a `<see cref="SignedHttpRequestClaimTypes.Ts"/>` in the comment.